core/imageroot/usr/local/bin/certificate-exporter

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (C) 2021 Nethesis S.r.l.
+# http://www.nethesis.it - nethserver@nethesis.it
+#
+# This script is part of NethServer.
+#
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see COPYING.
+#
+
+#
+# The script parses a traefik acme.json file and search for
+# Let's Encrypt certificate.
+# If a certificate has been created or updated,
+# certificate and private keys are saved inside Redis in key
+# named module/<module_id>/certificate/<domain> and
+# the certificate-update event is signaled.
+#
+
+import os
+import json
+import json
+import agent
+import os.path
+import argparse
+
+def _read_and_set(path, module_id, verbose):
+    base_key = f'module/{args.module_id}/certificate'
+    rdb = agent.redis_connect(privileged=True)
+    with open(path) as fp:
```

Comment:
A non-interactive script could expect to run with the correct `WorkingDirectory=`

```suggestion
    with open('./acme.json') as fp:
```